app/src/views/TermsAndConditions/index.tsx

```diff
@@ -116,35 +135,36 @@ export function Component() {
                 <Container
                     heading={strings.dataProvenanceHeading}
                     withHeaderBorder
-                    containerRef={dataProvenanceRef}
+                    elementId={sectionUrlHash.dataProvenance}
                 >
                     {strings.dataProvenanceDescription}
                 </Container>
                 <Container
                     heading={strings.userRegistrationHeading}
                     withHeaderBorder
-                    containerRef={userRegistrationRef}
+                    elementId={sectionUrlHash.userRegistration}
                     contentViewType="vertical"
                 >
                     <div>{strings.userRegistrationDescription}</div>
-                    <div>
-                        {resolveToComponent(
-                            strings.userRegistrationDescriptionToLink,
-                            {
-                                termsLink: (
-                                    <Link
-                                        to="register"
-                                        withUnderline
-                                    >
-                                        {strings.userRegistrationDescriptionLink}
-                                    </Link>
-                                ),
-                            },
-                        )}
-                    </div>
+                    {!isAuthenticated && (
+                        <div>
```

Comment:
This div is not needed.

Reply:
The div is there to insure the paragraph break